### ISSUE TITLE
Strip leading 0x from a raw tx string when sending an observation request

### ISSUE THREAD

**Issue by SEJeff** [wormhole-foundation/wormhole#2142]
On occasion, a request needs to be manually reobserved. In this instance, it is useful to just copy the tx straight from an explorer into the commandline and execute it.

## Expected Behavior

When running a send-observation-request admin command, if the raw tx is pasted in, it should just strip the leading `0x` instead of 🤮.

## Current Behavior

```bash
$ guardiand admin --socket /run/guardiand/admin.socket send-observation-request 2 0x9588802953bdc53ea3a60dc66a7d256a7631d2d0a3f5ef30a45bc4c026a9bd14
2022/12/16 14:47:59 invalid transaction hash (neither hex nor base58): invalid base58 digit ('0')
```

It works as expected when the leading `0x` is stripped off.

## Possible Solution

Find the right place to use [strings.TrimPrefix()](https://pkg.go.dev/strings#TrimPrefix) on the raw tx string and write unit tests for it.

**Comment by miikkuu** [comment]
Would like to work on this!

**Comment by SEJeff** [comment]
Ah sorry, this was fixed in #2274 though we would love your help in other areas @miikkuu
